src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        try {
+            return actorDao.add(actor);
+        } catch (Exception e) {
+            throw new DataProcessingException("Failed to add to the Db actor: " + actor);
+        }
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id)
+                .orElseThrow(() -> new DataProcessingException("Actor not found with id: " + id));
```

Comment:
Better to use different exceptions when there are problems with DB (DataProcessing) and when everything is ok and we just received request to find something we don't have in DB (EntityNotFound)
```suggestion
        return actorDao.get(id)
                .orElseThrow(() -> new EntityNotFoundException("Actor not found with id: " + id));
```